src/postprocess/chi0.jl

```diff
@@ -153,9 +153,9 @@ function apply_χ0(ham, δV, ψ, occupation, εF, eigenvalues; droptol=0,
     end
 
     # Add variation wrt εF
-    if model.temperature > 0
-        ldos = LDOS(εF, basis, eigenvalues, ψ)
-        dos = DOS(εF, basis, eigenvalues)
+    if temperature > 0
+        ldos = LDOS(εF, basis, eigenvalues, ψ, T=temperature)
+        dos = DOS(εF, basis, eigenvalues, T=temperature)
```

Comment:
homogenize the `T` argument as temperature over there as well?

Reply:
good idea. That's been bothering me for quite some time.